standards/SPIFFE_Workload_API.md

```diff
@@ -360,6 +366,122 @@ When interfacing with legacy JWT validators, the `FetchJWTBundles` method can be
 
 The `FetchJWTBundles` method returns bundles keyed by the SPIFFE ID of the trust domain. When validating a JWT-SVID, the validator MUST use the bundle corresponding to the trust domain of the subject. If a JWT bundle for the specified trust domain is not present, then the token is untrusted.
 
+## 7. WIT-SVID Profile
+
+The WIT-SVID Profile of the SPIFFE Workload API provides a set of gRPC methods which can be used by workloads to retrieve [WIT-SVIDs](WIT-SVID.md) and their related trust bundles. This profile outlines the signature of these methods, as well as related client and server behavior.
+
+Unlike the X.509-SVID and JWT-SVID profiles, implementation of the WIT-SVID profile is optional. Where a server does not support the WIT-SVID profile, calls to all RPCs defined by the WIT-SVID profile MUST return the "Unimplemented" gRPC status code. If the client receives an "Unimplemented" status code calling a WIT-SVID profile RPC, it SHOULD NOT attempt to reconnect to the server using the WIT-SVID profile.
+
+### 7.1 Profile Definition
+
+The WIT-SVID Profile RPCs and associated messages are defined below. For the complete Workload API service definition, see [workloadapi.proto](workloadapi.proto).
+
+```protobuf
+service SpiffeWorkloadAPI {
```

Comment:
I took a re-read of the SPIFFE Workload Endpoint specification - and within that - we do recommend that extensions to the endpoint be achieved by adding new gRPC services rather than modifying existing ones. One of the benefits claimed is that it makes it clear via gRPC reflection whether or not functionality is supported. This has me wondering if we should consider implementing the WIT-SVID profile as its own gRPC service. WDYT?

Reply:
I think that's mostly about adding new functionality that is not port of the WorkloadAPI, so I think we should keep this profile as part of the workloadapi service.

Reply:
> Aka discoverable by reflection?

This is an interesting idea to me and I can see the merit in it, but, I do think this could be awkward in some ways. Today, an implementation of a SPIFFE Workload API in Go can directly use the generated Go Protobuf bindings from the `go-spiffe` repository. However, if we are to say that you should only use the protobuf containing these if you implement the service, then they must generate their own Go Protobuf bindings (unless we were to publish a generated binding with/without the RPCs).